app/presenters/migration/index.rb

```diff
@@ -5,16 +5,56 @@ class Index < ApplicationPresenter
     def content
       mustache[:content] ||= begin
         {
-          title: page_content_heading
+          channel_info: {
+            name: page_content_heading
+          },
+          title: page_content_heading,
+          hero: {
+            url:              '/images/channel_hero_migrations.jpg',
```

Comment:
The images in this branch need to be moved from public/images to assets/images, then these references to them constructed with: `asset_path('channel_hero_migrations.jpg')`